Services/User/classes/class.ilPublicUserProfileGUI.php

```diff
@@ -152,17 +152,17 @@ public function executeCommand(): string
 
         $tpl->loadStandardTemplate();
 
+        if($next_class == "ilobjportfoliogui" && $cmd != "deliverVCard") {
```

Comment:
```suggestion
        if ($next_class === strtolower(ilObjPortfolioGUI::class) && $cmd !== 'deliverVCard') {
```

I always suggest using the `class-string` instead. If the class is renamed in future, modern IDEs will find usages like this more reliable.

Reply:
I do not think this is relevant in showing the issue here. this is not intended for merging. please see the related ticket.